### DISCUSSION TITLE
最近保存，总是重复了

### DISCUSSION THREAD

**Discussion by huangjingyun8** [coddingtonbear/obsidian-web#175] [Bug Report]
### Describe the bug

1.按之前的操作，显示保存成功后，换下一个再操作一次保存，又一次显示成功。

2.在 日记里，会显示如下，两条一模一样了，这种情况已经延续好几天了。

<img width="713" alt="image" src="https://github.com/coddingtonbear/obsidian-web/assets/56347021/614f4273-bb1c-4adc-bbd3-632fa189d837">

3.不得不再来拜托🙏大佬

### To Reproduce

2. Click on '...'
3. Scroll down to '...'

### Operating System

mac

### Browser

chrome

### Browser Version

122.0.6261.94（正式版本） (x86_64)

**Comment by coddingtonbear** [comment]
I'm afraid I don't speak mandarin and the translation isn't making anything very clear to me -- is there any way you might be able to elaborate a little more about what kind of problem you're encountering?

**Reply by huangjingyun8** [reply]
sorry，Last time I mentioned bugs, and you identified it and solved it, I thought you knew Chinese.

`On the X.com homepage, click Save Different Information Flows will   save the same information twice.`

**Comment by coddingtonbear** [comment]
Saving the content twice if you press the button twice is what I would expect if your template is set up to append content to whatever note (that is, that your template is using a `POST` or `PUT` method).  I’m curious to hear what you might have been expecting, though, so I can give you suggestion about how you might update your template to better map to what you are looking for.

**Reply by huangjingyun8** [reply]
![image](https://github.com/coddingtonbear/obsidian-web/assets/56347021/9b46983b-bba0-4a5e-9d59-1a0235db908d)

I  always  save it  with
Append to current daily note

I now have to check every time after saving to see if there are duplicates.

Now, I click A to save, click B to save, and 2 A's will be saved in the obsidian diary. I want to restore it like before. In the A information, I click save, and then only save A once in the diary.

**Reply by huangjingyun8** [reply]
Now, if I click save A and I check Obsidian's daily diary, A is not saved, but then I click save B and check again, Obsidian saved two A's. I don't know what the logic is?

**Reply by huangjingyun8** [reply]
Now I found a problem, I am using obsidian web on arc for mac and it works. But when used on chrome for mac, it won't work.

**Comment by coddingtonbear** [comment]
"Append" means "add to the bottom of".  Assuming you have not modified your "Append to current daily note" template, that should always add your clipped content to the bottom of your daily note every time you press the button.  So, I'm afraid the behavior you're describing in the beginning of this discussion is the expected behavior and I'll be closing this issue so as to limit the noise in Discussions section.

If you're instead experiencing Obsidian Web intermittently *not* sending content to your notes at all, please go to #172 and see if you can help us figure out how to reproduce that problem.  I have so far personally not been able to reproduce this problem no matter how many times I try to do so -- the content I want to clip always appears the very first time I send the content -- and that means that it's very likely that you and the others who are experiencing that problem are using Obsidian Web in a slightly different way than I'm expecting.  So: I'm hoping one of the folks experiencing this bug can produce more detailed reproduction steps or a screencast demonstrating the problem so I can find the difference.

**Reply by huangjingyun8** [reply]
Thank you so much
